flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -171,6 +171,11 @@ class SentryFlutterOptions extends SentryOptions {
   /// [SentryFlutter.setAppStartEnd].
   bool autoAppStart = true;
 
+  /// Enable capturing CPU profiles alongside transactions.
+  /// Only available on iOS and Android. This is currently a beta feature that is
+  /// not available to all Sentry customers.
```

Comment:
I'd also write that transactions started on the Dart layer won't have profile data, but only transactions started on the Native layer (Android/iOS).